package/cfg/rke2-cis-1.23-hardened/policies.yaml

```
@@ -183,7 +183,7 @@ groups:
 
       - id: 5.2.8
         text: "Minimize the admission of containers with the NET_RAW capability (Automated)"
-        audit: "kubectl get psp global-restricted-psp -o json | jq -r .spec.requiredDropCapabilities[]"
+        audit: "kubectl get psp global-restricted-psp && kubectl get psp global-restricted-psp -o json | jq -r .spec.requiredDropCapabilities[] || kubectl get psp restricted-psp && kubectl get psp restricted-psp -o json | jq -r .spec.requiredDropCapabilities[] || kubectl get psp unrestricted-psp && kubectl get psp unrestricted-psp -o json | jq -r .spec.requiredDropCapabilities[] || kubectl get psp restricted-noroot-psp -o json | jq -r .spec.requiredDropCapabilities[]"
```

same question as the above comment. 

```suggestion
        audit: "kubectl get psp global-restricted-psp && kubectl get psp global-restricted-psp -o json | jq -r .spec.requiredDropCapabilities[] || kubectl get psp restricted-psp && kubectl get psp restricted-psp -o json | jq -r .spec.requiredDropCapabilities[] || kubectl get psp unrestricted-psp && kubectl get psp unrestricted-psp -o json | jq -r .spec.requiredDropCapabilities[] || kubectl get psp restricted-noroot-psp && kubectl get psp restricted-noroot-psp -o json | jq -r .spec.requiredDropCapabilities[]"
```